Makefile

```diff
@@ -291,10 +291,15 @@ image-test: image-clang
 	$(QUIET)@echo "Push like this when ready:"
 	$(QUIET)@echo "${CONTAINER_ENGINE} push cilium/tetragon-test:$(DOCKER_IMAGE_TAG)"
 
-image-codegen:
-	$(CONTAINER_ENGINE) build -f Dockerfile.codegen -t "cilium/tetragon-codegen:${DOCKER_IMAGE_TAG}" .
+image-protogen:
+	$(CONTAINER_ENGINE) build -f Dockerfile.protogen -t "cilium/tetragon-protogen:${DOCKER_IMAGE_TAG}" .
 	$(QUIET)@echo "Push like this when ready:"
-	$(QUIET)@echo "${CONTAINER_ENGINE} push cilium/tetragon-codegen:$(DOCKER_IMAGE_TAG)"
+	$(QUIET)@echo "${CONTAINER_ENGINE} push cilium/tetragon-protogen:$(DOCKER_IMAGE_TAG)"
+
+image-crdgen:
+	$(CONTAINER_ENGINE) build -f Dockerfile.crdgen -t "cilium/tetragon-crdgen:${DOCKER_IMAGE_TAG}" .
```

Comment:
I think your `Dockerfile.crdgen` is maybe not useful as you can just use the `docker.io/library/golang:1.22.0@sha256:7b297d9abee021bab9046e492506b3c2da8a3722cbf301653186545ecc1e00bb` image with a `docker run --env myenv`.